app/services/proofing/lexis_nexis/ddp/proofer.rb

```diff
@@ -14,7 +14,9 @@ class Proofer < LexisNexis::Proofer
                             :address1,
                             :city,
                             :state,
-                            :zipcode
+                            :zipcode,
+                            :input_ip_address,
+                            :partner_sp_app_id
```

Comment:
we call this `uuid_prefix` in other places, I'd recommend using that:

https://github.com/18F/identity-idp/blob/7dbd3ef381b6737cfd291e15c0d264fa9f962ca9/app/services/proofing/mock/address_mock_client.rb#L8

https://github.com/18F/identity-idp/blob/effb3eb3531ae60c98bb466c155b5bea13eb054e/app/services/proofing/lexis_nexis/request.rb#L82-L90

Reply:
That also means we *should* be already passing in the uuid prefix inside the `applicant`, so in the `ResolutionProofingJob` we could add an assertion to make sure it's there too

Reply:
You mean in the spec here? https://github.com/18F/identity-idp/blob/effb3eb3531ae60c98bb466c155b5bea13eb054e/spec/jobs/resolution_proofing_job_spec.rb#L46

Reply:
not just in the spec! in the actual implementation as well